db/psychologists.ts

```diff
@@ -44,44 +44,56 @@ const saveAssignedUniversity = async (psychologistId: string, assignedUniversity
   return updatedPsy;
 };
 
+const allActivePsychologistRequest = db.select([
```

Comment:
:thinking: But I don't want this part to be executed when called. This is just to avoid duplicating this code between the 2 functions. But in one case, I want to execute this request, in another case, I want to append more "conditions". 
Using a function here will return the psychologists and I don't want that.

Do you see any better way than duplicating this piece of code?